burnman/seismic.py

```diff
@@ -193,27 +226,39 @@ class SeismicRadiusTable(Seismic1DModel):
     sorted by radius. Fill the tables in the constructor after deriving
     from this class. This class uses :class:`burnman.seismic.Seismic1DModel`
     
-    Note: all tables need to be sorted by increasing radius.
+    Note: all tables need to be sorted by increasing depth or radius.
     Alternatively, you can also overwrite the _lookup function if you
     want to access with something else.
     """ 
     def __init__(self):
         Seismic1DModel.__init__(self)
+        self.table_depth = []
         self.table_radius = []
         self.table_pressure = []
+        self.table_gravity = []
         self.table_density = []
         self.table_vp = []
         self.table_vs = []
+        self.table_QG = []
+        self.table_QK = []
+
         self.earth_radius = 6371.0e3
         
-    def internal_depth_list(self):
-        
-        return (self.earth_radius - self.table_radius)[::-1] #radius is sorted in increasing order, so we need to reverse the depth list
+    def internal_depth_list(self,mindepth=0., maxdepth=6371.e3):
+        if not len(self.table_depth) == 0:
+            return np.array([self.table_depth[x] for x in range(len(self.table_depth)) if self.table_depth[x]>mindepth and self.table_depth[x]<maxdepth]) 
+        else:
+            depth = self.earth_radius - self.table_radius[::-1]
+            return np.array([depth[x] for x in range(len(depth)) if depth[x]>mindepth and depth[x]<maxdepth])
```

Comment:
instead of creating this whole array every time this function is called, can you put the formula earth_radius-radius inside the ```[ ... ]```?